index.html

```diff
@@ -307,19 +323,19 @@ <h2 class="subtitle">We are always in touch we the outside world</h2>
         <div class="tabs is-boxed is-centered main-menu" id="nav">
```

Comment:
In this block, I suggest getting rid of **<li**
```suggestion
       <div class="tabs is-boxed is-centered main-menu" id="nav">
          <div class="tablist" role="tablist" aria-label="tablist picture, team and video">
            <button class="is-active" role="tab" aria-selected="true" aria-controls="pane-1" data-target="pane-1" id="1" >
                <span class="icon is-small"><i aria-hidden="true"class="fa fa-image"></i></span>
                <span>Pictures</span>
            </button>
            <button role="tab" tabindex="-1" aria-selected="false" aria-controls="pane-3" id="3" data-target="pane-3">
                <span class="icon is-small"><i aria-hidden="true" class="fab fa-superpowers"></i></span>
                <span>Team</span>
            </button>
            <button role="tab" tabindex="-1" aria-selected="false" aria-controls="pane-4" data-target="pane-4" id="4" data-deletable=""> 
                <span class="icon is-small"><i aria-hidden="true" class="fa fa-film"></i></span>
                <span>Video</span>
            </button>
          </div>
        </div>
```